site/includes/headresources.hbs

```diff
@@ -1,6 +1,6 @@
 <!--[if gte IE 9 | !IE ]><!-->
 <link href="{{assets}}/../{{site.theme}}/assets/favicon.ico" rel="icon" type="image/x-icon" />
-<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-Mmxa0mLqhmOeaE8vgOSbKacftZcsNYDjQzuCOm6D02luYSzBG8vpaOykv9lFQ51Y" crossorigin="anonymous" />
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```

Comment:
```suggestion
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous" />
```
Add self-closing slash.

Reply:
Ah thanks @EricDunsworth  - yeah I just copied from the fontawesome site again so lost that.